galasa-ui/src/components/test-runs/TestRunsTable.tsx

```diff
@@ -64,11 +64,27 @@ export default function TestRunsTable({runsList,limitExceeded, visibleColumns, o
   const [currentPage, setCurrentPage] = useState(1);
   const [pageSize, setPageSize] = useState(10);
 
+  const [showNotification, setShowNotification] = useState(limitExceeded);
+  
   const headers = orderedHeaders?.filter(column => visibleColumns.includes(column.id)).map(column => ({
     key: column.id,
     header: translations(column.id)
   })) || [];
 
+  // Set a timer to hide the notification after 5 seconds
+  useEffect(() => {
+    // Only show the notification if the limit was exceeded
+    if (limitExceeded) {
+      setShowNotification(true);
+      const timer = setTimeout(() => {
+        setShowNotification(false);
+      }, WARNING_NOTIFICATION_DURATION);
+
+      // Cleanup function: This will clear the timer unmounts before the timeout.
+      return () => clearTimeout(timer);
```

Comment:
This code is duplicated. Might be worth defining a common function to show the notification and set the un-show up at the same time ?

We could then use the same thing for info notifications also (like "link has been copied to the clipboard")...